Frameworks/UIKit/UITextField.mm

```diff
@@ -213,12 +224,73 @@ - (NSDictionary*)defaultTextAttributes {
 }
 
 /**
- @Status Caveat
- @Notes Fonts can be set, but will be ignored for rendering, instead default font will be used.
+ @Status Interoperable
 */
 - (void)setFont:(UIFont*)font {
-    _font = font;
-    // TODO 7374333: need map UIFont with fontFamily/FontSize/FontWeight/FontStyle on target
+    if (font == nil) {
+        _font = [UIFont fontWithName:@"Segoe UI" size:[UIFont labelFontSize]];
+    } else {
+        _font = font;
+    }
+    [self _adjustFontSizeToFitWidthOrApply];
+}
+
+- (void)_applyFont:(UIFont*)font {
+    _passwordBox.fontFamily = _textBox.fontFamily = [WUXMFontFamily makeInstanceWithName:[font _compatibleFamilyName]];
+
+    // The following enums map from DWrite directly to Xaml
+    _passwordBox.fontStretch = _textBox.fontStretch = (WUTFontStretch)[font _fontStretch]; 
+    _passwordBox.fontStyle = _textBox.fontStyle = (WUTFontStyle)[font _fontStyle]; 
+    _passwordBox.fontSize = _textBox.fontSize = font.pointSize;
+
+    WUTFontWeight* weight = nil;
+
+    switch ([font _fontWeight]) {
+        case DWRITE_FONT_WEIGHT_THIN:
+            weight = [WUTFontWeights thin];
+            break;
+        //case DWRITE_FONT_WEIGHT_EXTRA_LIGHT:
```

Comment:
Why are these commented out here - are they applicable in some way. The code will fall into the default block for these values so why not make them as a comment there.